composer.lock

```diff
@@ -3849,5 +3866,6 @@
         "php": ">=7.1",
         "ext-mbstring": "*"
     },
-    "platform-dev": []
+    "platform-dev": [],
+    "plugin-api-version": "2.0.0"
```

Comment:
WARNING for shipping to production

We should check whether Prestonbot is deployed using Composer v1 or Composer v2

Reply:
Not a problem, it's using `"plugin-api-version": "1.0.0"` with composer v2 that is the problem ;)